free-2.md

```diff
@@ -11,41 +11,118 @@ _What need is met by your idea? Who are you helping? What is that person's
 experience like now? What would their experience be like if you could help 
 them?_
 
+Mudders get a lot of email. 
+Many people subscribe to dorm chats and occasionally receive
+email with interesting content but a lot of times they get useless
+emails (these range from counting threads, to emails asking for rides from
+airports, and to ASHMC senate summary emails).
+
+As an example, these are a series of emails sent to a certain dorm's chat
+list
```

Comment:
THIS IS AMAZING!!! As someone who gets TOnNNSSS of horrible emails (I admit that I do partake in counting threads ...) i personally find a good use in this. I am definitely someone you would be helping so I relate to your picture.